app/webpacker/css/admin/products_v3.scss

```diff
@@ -170,12 +170,6 @@
     .field {
       padding: 0;
     }
-    .multi-field {
-      // Allow wrap with small gap
-      display: flex;
-      flex-wrap: wrap;
-      gap: 3px;
-    }
```

Comment:
Ah sorry about that I forgot to leave a note here.
> I'm guessing because there was an issue when the row is wider, with the grey background showing through at the bottom

Yes, exactly. Then I used the `field` class and I thought the existing style should be fixed. But yes, indeed, I must have broken this, sorry about that.:
> This style rule was to maintain the gap between the dropdown, and the extra input that appears when you select "Items". If it is removed, I think it will sit too close again.

Reply:
>so made this rule instead

Great, that's the suggestion ☺

Reply:
It's updated here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12401/commits/2c36dd20e79339b96c8220e5332eb2dbd1cf6bb0